src/test/java/com/clouway/push/adapter/json/JsonEventSerializerTest.java

```diff
@@ -0,0 +1,37 @@
+//package com.clouway.push.adapter.json;
+//
+//import com.clouway.push.core.PushEventSource;
+//import com.clouway.push.testevents.AddPersonEvent;
+//import com.clouway.push.testevents.RemovePersonEvent;
+//import org.junit.Test;
+//
+//import static org.hamcrest.Matchers.equalTo;
+//import static org.hamcrest.Matchers.is;
+//import static org.junit.Assert.*;
+//
+//
+///**
+// * @author Miroslav Genov (miroslav.genov@clouway.com)
+// */
+//public class JsonEventSerializerTest {
```

Comment:
Commented class.

Reply:
My mistake. I added a property and forgot about the test. Will be fixed